script.js

```diff
@@ -0,0 +1,18 @@
+const hamburger = document.querySelector('.fa-bars');
+const MBMenu = document.querySelector('.mb_menu');
+const cross = document.querySelector('#x_icon');
+const menuItems = document.getElementsByClassName('menu_list_items');
```

Comment:
- [ OPTIONAL ] I would recommend using `querrySelectorAll` to query elements from the `DOM` as It offers better performance when using complex queries or selecting multiple elements ⚡.